app/controllers/results_controller.rb

```diff
@@ -60,8 +61,8 @@ def edit
     reviewer_access = false
     if @result.is_a_review?
       if @current_user.is_reviewer_for?(@assignment.pr_assignment, @result)
+        @assignment = @assignment.pr_assignment
```

Comment:
Don't change `@assignment`. I don't want to have to worry about what this refers to.